lib/dxf/parser.rb

```diff
@@ -90,15 +96,14 @@ def parse_entities(io)
       parser = nil
       parse_pairs io do |code, value|
         if 0 == code.to_i
-          if parser
+          if parser && value != 'VERTEX'
```

Comment:
the Vertex entity is included to prevent the parser from jumping to the next object (as Vertex is a subclass of Polyline and part of it)